src/domain/states/objects/fileTree/SchemaFile.ts

```diff
@@ -11,6 +11,7 @@ export class SchemaFile extends SchemaTreeItem {
         public readonly schemaFile: string,
         public readonly dataFile: string,
         variants: SchemaFileVariant[] = [],
+        public selected: boolean = false,
```

Comment:
pls remove this, it doesn't belong here. See also comment in FileTree.tsx

Reply:
done